src/libnetdata/socket/security.h

```diff
@@ -14,6 +14,16 @@ typedef enum __attribute__((packed)) {
 
 # ifdef ENABLE_HTTPS
 
+#if defined(ENABLE_WOLFSSL)
+#include <wolfssl/options.h>
+#include <wolfssl/version.h>
+#include <wolfssl/ssl.h>
+#include <wolfssl/error-ssl.h>
+
+#include <wolfssl/openssl/ssl.h>
+#include <wolfssl/openssl/sha.h>
+#include <wolfssl/openssl/evp.h>
+#elif defined(ENABLE_OPENSSL)
```

Comment:
Can we create a separate header that will abstract away the SSL vendor and include the proper things there? Something like libnetdata/ssl/ssl.h